source/Plugins/RiotLibraryCore/RiotLibrary.cs

```diff
@@ -0,0 +1,146 @@
+﻿using RiotLibrary.Models;
+using Playnite.Common;
+using Playnite.SDK;
+using Playnite.SDK.Models;
+using Playnite.SDK.Plugins;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace RiotLibrary
+{
+    public class RiotLibrary : LibraryPlugin
+    {
+        private ILogger logger = LogManager.GetLogger();
+        private const string dbImportMessageId = "bnetlibImportError";
+
+        internal RiotLibrarySettings LibrarySettings { get; private set; }
+
+        public RiotLibrary(IPlayniteAPI api) : base(api)
+        {
+            LibrarySettings = new RiotLibrarySettings(this, PlayniteApi);
+        }
+
+        public static UninstallProgram GetUninstallEntry(RiotApp app)
+        {
+            foreach (var prog in Programs.GetUnistallProgramsList())
+            {
+
+                if (string.IsNullOrEmpty(prog.UninstallString))
+                {
+                    continue;
+                }
+
+                var match = Regex.Match(prog.UninstallString, @"ThisWillNeverMatch");
```

Comment:
if this never will match why we need it?

Reply:
Lol, that was a test I was doing, to remove it later
and I forgot